app/schema/get-search-api-data-schema.tsx

```diff
@@ -0,0 +1,11 @@
+import z from "zod";
+
+const SearchSchema = z.object({
+  results: z.array(
+    z.object({
+      year: z.optional(z.string()),
+    })
+  ),
+});
+
+export default SearchSchema;
```

Comment:
I'd make this a named export instead

Reply:
What's the significance of this? Updated to named export.

Reply:
if a file has a very specific use with one export (a single function, component, etc.) then you want to use a default export. If it's a more generic file like this one that will have multiple exports, typically you'll want to use named exports instead.